src/_base/harness/config/pipeline.yml

```
@@ -88,6 +88,9 @@ command('app deploy <environment>'):
     set -o pipefail
     cd "${ENVIRONMENT}"
     doctl -t "$DO_ACCESS_TOKEN" kubernetes cluster kubeconfig show "$CLUSTER" > kubectl.config.yaml
+    if helm version --short --client | grep '^v3' >/dev/null 2>&1; then
+      chmod go-r kubectl.config.yaml
+    fi
```

```suggestion
    (umask 0200 && doctl -t "$DO_ACCESS_TOKEN" kubernetes cluster kubeconfig show "$CLUSTER" > kubectl.config.yaml)
```

I think its fine making it for helm v2 as well. This makes it so the file never got go+r

---

Yeah, I think helm2 would be ok with this as well. Thanks! 🙏